docs/management/advanced-options.asciidoc

```diff
@@ -142,7 +142,14 @@ This setting does not have an effect when loading a saved search.
 Highlighting slows requests when
 working on big documents.
 
+[float]
+[[kibana-ml-settings]]
+==== Machine learning
 
+[horizontal]
+`ml:fileDataVisualizerMaxFileSize`:: Sets the file size limit when importing
```

Comment:
If you accept my suggestion about changing the setting name, it will need to be updated here too:
```suggestion
`ml:DataVisualizerMaxFileSize`:: Sets the file size limit when importing
```